src/foo.yml

```
@@ -3,7 +3,7 @@ invoice: 34843
 date   : 2001-01-23
 bill-to: &id001
     given  : Chris
-    family : Dumars
+    family : Doe
```

has he changed the surname to "Doe"?
```suggestion
    family : Døe
```
I think this is correct ^^^